websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -269,24 +282,19 @@ Icons on **narrow** cards are aligned to the left of the card regardless of whet
 
 ```jsx
 <Cards>
-<Card title="Bullet List Body">
-
-* Item 1
-* Item 2 (dot)
-  * Sub-item (circle)
-* Item 3
-
-</Card>
+  <Card title="Bullet List Body">
+    * Item 1 * Item 2 (dot) * Sub-item (circle) * Item 3
+  </Card>
```

Comment:
whoo, what was going wrong here.  Some code formatter decided to not handle the example block as markdown.   We'll have to switch the tags from `jsx` to `mdx` or `md`